course-v2/layouts/partials/get_instructors.html

```diff
@@ -4,17 +4,15 @@
   {{ $url := (print (strings.TrimSuffix "/" $staticApiBaseUrl) "/instructors/" . "/index.json") }}
   {{ with resources.GetRemote $url }}
     {{ with .Err }}
-      {{ $errorMessage := printf "Something went wrong while fetching instructors on %v via %v with error: %v" site.BaseURL $url .Err }}
-      {{ partial "sentry_capture_message.html" $errorMessage }}
+      {{ errorf "Something went wrong while fetching instructors on %v via %v with error: %v" site.BaseURL $url . }}
     {{ else }}
       {{ $data := (. | unmarshal).data }}
       {{ $searchUrl := partial "get_search_url.html" (dict "key" "instructors" "value" (title $data.title)) }}
       {{ $instructor := merge $data (dict "url" $searchUrl) }}
       {{ $instructors = $instructors | append $instructor }}
     {{ end }}
   {{ else }}
-    {{ $errorMessage := printf "Failed to fetch course instructors through %v on %v" $url site.BaseURL }}
-    {{ partial "sentry_capture_message.html" $errorMessage }}
+    {{ errorf "Failed to fetch course instructors through %v on %v" $url site.BaseURL }}
```

Comment:
This block is reached if Hugo successfully makes the HTTP call, but the response is 404. Hugo treats such cases as "empty", not as errors. [^1]

[^1]: @gumaerc @pt2302 does my description jive with your understanding?

**This is a behavior change:** Previously, we did nothing[^2] when this block was hit. Now we fail the build.

[^2]: We were trying to alert sentry. But we were unsuccessful because, again, partials cannot both return a value and render markup.